src/options/options.html

```diff
@@ -90,6 +90,11 @@ <h6 class="vocab heading">Store and load the last search results when you open t
             <input type="checkbox" class="vocab choice-field magenta-colored small-sized" id="enable-search-storage-checkbox" />
             <label for="enable-search-storage-checkbox" class="padding-left-smaller">Yes</label><br />
           </div>
+          <h6 class="vocab heading margin-top-medium">Ask for confirmation everytime before clearing search result?</h6>
+          <div class="enable-search-storage-container">
```

Comment:
The class should not be `enable-search-storage-container` as this is not the search storage checkbox.

If you look at the `enable-search-storage-container` [rule in _options.scss](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/options/sass/components/_options.scss#L17), you will find that it only has `margin-left: 1rem;` property. 

We can do better than this.
- Remove this class from this and the above div and also remove the rule from `_options.scss`. 
- Define a utility class, `margin-left-medium` in [_utilities.scss](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/options/sass/base/_utilities.scss) and use that on both the divs.